exit.go

```diff
@@ -149,49 +120,26 @@ func (te *TunaExit) handleSession(session *smux.Session, conn net.Conn) {
 			if te.config.Reverse {
 				continue
 			}
-			go func(stream *smux.Stream) {
-				txData, err := ioutil.ReadAll(stream)
-				if err != nil && err.Error() != io.EOF.Error() {
-					log.Println("Couldn't read payment stream:", err)
-					return
-				}
-				if len(txData) == 0 {
-					return
-				}
-				tx := new(transaction.Transaction)
-				if err := tx.Unmarshal(txData); err != nil {
-					log.Println("Couldn't unmarshal payment stream data:", err)
-					return
+			totalCost := common.Fixed64(0)
```

Comment:
nanoPayClaim might be blocking because it has RPC call. It's better to put this part in goroutine like before.